### DISCUSSION TITLE
How to get a response with hyperlinks?

### DISCUSSION THREAD

**Discussion by home-apps** [FlowiseAI/Flowise#329] [Q&A]
The data loaded contains text and it includes a list of links. How can I get chat response with clickable hyperlinks? 

Example:

"to answer your question, these are the recommended resources:

https://resource1.com
https://resource2.net

**Comment by HenryHengZJ** [comment]
will be working on it

**Comment by chungyau97** [comment]
[Flowise PR #671](https://github.com/FlowiseAI/Flowise/pull/671)
[FlowiseChatEmbed PR #24](https://github.com/FlowiseAI/FlowiseChatEmbed/pull/24)
